src/r_voxel.c

```diff
@@ -317,7 +317,7 @@ void VX_Init (void)
 
 //------------------------------------------------------------------------
 
-#define VX_MINZ         (   1 * FRACUNIT)
+#define VX_MINZ         (   4 * FRACUNIT)
```

Comment:
This eliminates flickering for me. But maybe clipping is less effective, or it depends on the resolution? I am not sure.

Reply:
That change fixes flickering for me too, but I'll admit I'm not familiar with the voxel code.

Reply:
I've tested it further, it doesn't seem to affect performance and I don't see any problems. I've taken this value from the original `R_ProjectSprite`: https://github.com/fabiangreffrath/woof/blob/c29b89cae2a11805a1e743717e872c21aa284adc/src/r_things.c#L36
@fabiangreffrath Should we merge this?

Reply:
Reasonable. Yes, please merge. Thank you!